docs/product-assembly_dm-on-openshift/src/main/asciidoc/installing-images-file-proc.adoc

```diff
@@ -0,0 +1,16 @@
+[id='installing-images-file-proc']
+= Installing the image streams file
```

Comment:
I'll mention this only here, but it applies to basically all of your procedure modules. All proc modules should have the title, context statement, .Procedure heading, then numbered steps or a single unnumbered step if only one (and if needed, a bulleted .Prerequisites section before .Procedure). As you know from the [module templates](https://mojo.redhat.com/docs/DOC-1099486#jive_content_id_Master_source_files_in_asciidoc_format). You do an excellent job using consistent and meaningful context statements, but after that, there's not usually a .Procedure and steps or step in your proc modules, and not much structure in general, imo. It's important to try to follow that format as much as possible, not just for the sake of doing it or being consistent (which are indeed important), but also because it dramatically improves readability/scannability of the doc. Users can quickly skim and see procedure steps or step and buzz through readily, without having to think or process so much.

Here's my whack at this section, for example. This is a good instance of an unnumbered one-step procedure. Others with multiple steps would be numbered (duh).

Installing the image streams file

The `image_streams.json` file in {PRODUCT} provides the required information for downloading necessary images from the Red Hat repository. You must install this file into the OpenShift environment in order to properly deploy Decision Manager on OpenShift.

.Procedure
Install the `image_streams.json` file from either the OpenShift web interface or the command line:
 * Using the OpenShift web interface, select **Add to Project -> Import YAML / JSON*, then choose the file or paste its contents.
 * Using the command line, enter the following command:
 [subs="verbatim,macros"]
--
 $ oc create -f image_streams.json
--

**<For either of these, is more context necessary as to where the file is found? In the first, if they need to know where to navigate to when selecting from the file options in the UI, and in the second, if they need to know where in the directory structure to enter that command.>**

**<Ignore my code snippet formatting. GitHub text limitation.>**

**<PS: "web" is lc, per IBM SG. Just for future reference.>**